### DISCUSSION TITLE
点击清空列表后再刷新列表，列表是空的

### DISCUSSION THREAD

**Discussion by smily1223** [MoeNetwork/Tieba-Cloud-Sign#186] [疑问]
因为有些贴吧失效。所以尝试清空列表。再刷新的时候就刷不出来了。只能手动添加贴吧

**Comment by smily1223** [comment]
![image](https://user-images.githubusercontent.com/37728519/187810355-e6f98aa8-c970-469d-89a7-b898bbd69aba.png)
手动手动添加贴吧也签到不了
已经重装过一次了。

**Comment by smily1223** [comment]
https://github.com/MoeNetwork/Tieba-Cloud-Sign/issues/113
根据这个回复，出现以下报错。
![image](https://user-images.githubusercontent.com/37728519/187811575-f7626e5b-7b93-4301-b19c-ed5f7c3205b3.png)

**Comment by BANKA2017** [comment]
#187

**Reply by smily1223** [reply]
> #187

我用的是最新的版本，而且我查了class.misc.php 是有该函数的
![image](https://user-images.githubusercontent.com/37728519/187823650-1d3c77d8-10ea-40f8-9f69-899af787f255.png)

**Reply by BANKA2017** [reply]
你只需要参照 #187 这个pr修改 `class.misc.php` 文件即可

**Reply by smily1223** [reply]
照#187这个改了，刷新贴吧列表还是不行，原来就是可以的，清空列表后不行的。

**Reply by smily1223** [reply]
改了后就test.php没有报错。刷新贴吧列表还是不行。

**Reply by BANKA2017** [reply]
> 刷新贴吧列表还是不行。

这里的 **不行** 指的是**不能刷新贴吧列表**（即点完**刷新贴吧列表**以后贴吧列表还是空白）还是**不能添加异常状态的贴吧**（比如被封掉的）？

如果可能的话，建议
- 打开开发者控制台检查一下 `http://域名/setting.php?mod=showtb&refreshTiebaList` 返回了什么
- 尝试更换`bduss`和`stoken`

**Reply by smily1223** [reply]
![image](https://user-images.githubusercontent.com/37728519/188043558-533d607a-f821-405e-b307-6037866dac72.png)
http://域名/setting.php?mod=showtb&refreshTiebaList这个返回的是主页面。
bduss和stoken也已经更换过。

实际情况是前2天看到有2个贴吧状态异常，就点了一下清空列表，列表清空后点刷新贴吧列表，贴吧列表就刷新不出来了。换了几次bduss和stoken，也照https://github.com/MoeNetwork/Tieba-Cloud-Sign/pull/187改过。手动添加贴吧能添加上，但是签到都是异常的，上面都有说。
![image](https://user-images.githubusercontent.com/37728519/188043924-9ba34c82-7a43-4f1d-a8b4-65b8b59b2a34.png)
